### ISSUE TITLE
Can an attendee share two cameras at once?

### ISSUE THREAD

**Issue by walkindude** [aws/amazon-chime-sdk-js#2855]
Hello, I'm investigating whether Chime SDK can fit the following scenario.

### What are you trying to do?

Picture a booth containing a remotely controlled medical device, and a computer with two USB cameras connected to it so that a remote doctor can see the patient from two angles: I want the doctor to be able to see the video coming from both cameras (and the patient should be able to see the doctor in turn, but I don't think that will be a problem). Of course one party should be able to hear the other's audio.

I can only imagine two scenarios of how this could happen, and both of them would be great:

1. The SDK allows me to have two video inputs for one attendee we'll call "booth"
2. The SDK allows me to create one attendee per camera and have both of them join the same meeting from within the same context (as in browser tab)

Is there a way to do this? I tried Googling, but I can't seem to find, either on the official docs or elsewhere, anything that tells me whether this is possible.

Many, many thanks in advance.

**Comment by ltrung** [comment]
Currently, our SDK only allows one local video for each attendee. If you need multiple video per attendee, please mark this as feature request.
For your use case, you can create different attendees to join the meeting. They can be in the same browser tab each with different local video.

**Comment by walkindude** [comment]
Thank you very much @ltrung. Creating one attendee per camera will work perfectly for me, and since there's a perfectly good way to do what I need, I won't flag this as a feature request.

Again thank you for the prompt answer.

**Comment by ByronBoskellWork** [comment]
Hello,
I am investigating the same scenario as walkindude. We would like to have multiple input cameras to show a remote doctor. 

The project is using the react component library. From my knowledge of the library it uses react hooks to provide functionality to its components. From my understanding if we did have two attendees in one tab, one would be ignored by all the components unless I rewrote all the components which use the hooks which is essentially the whole library. Since the chime sdk library would face the same issue I would like to make this a feature request.
